### ISSUE TITLE
Add Raspberry Pi Pico with MicroPython support

### ISSUE THREAD

**Issue by makermelissa** [adafruit/Adafruit_Python_PlatformDetect#162]
As noted in another issue: https://github.com/adafruit/Adafruit_Blinka/issues/445#issuecomment-821982501.

**Comment by makermelissa** [comment]
This should be an easy add after #161 gets merged
